src/data/localStorage.ts

```diff
@@ -8,46 +8,18 @@ const LOCAL_STORAGE_KEY_STATE = "excalidraw-state";
 const LOCAL_STORAGE_KEY_COLLAB = "excalidraw-collab";
 const LOCAL_STORAGE_KEY_LIBRARY = "excalidraw-library";
 
-let _LATEST_LIBRARY_ITEMS: LibraryItems | null = null;
-export const loadLibrary = (): Promise<LibraryItems> => {
-  return new Promise(async (resolve) => {
-    if (_LATEST_LIBRARY_ITEMS) {
-      return resolve(JSON.parse(JSON.stringify(_LATEST_LIBRARY_ITEMS)));
-    }
-
-    try {
-      const data = localStorage.getItem(LOCAL_STORAGE_KEY_LIBRARY);
-      if (!data) {
-        return resolve([]);
-      }
-
-      const items = (JSON.parse(data) as ExcalidrawElement[][]).map(
-        (elements) => restore(elements, null).elements,
-      ) as Mutable<LibraryItems>;
-
-      // clone to ensure we don't mutate the cached library elements in the app
-      _LATEST_LIBRARY_ITEMS = JSON.parse(JSON.stringify(items));
-
-      resolve(items);
-    } catch (e) {
-      console.error(e);
-      resolve([]);
-    }
-  });
+export const loadLibrary = async (): Promise<LibraryItems> => {
+  const data = localStorage.getItem(LOCAL_STORAGE_KEY_LIBRARY);
+  if (!data) {
+    return [];
+  }
+  return (JSON.parse(data) as ExcalidrawElement[][]).map(
+    (elements) => restore(elements, null).elements,
+  ) as Mutable<LibraryItems>;
```

Comment:
What's the idea behind not catching errors anymore? This will result in `loadedPromise()` promise downstream being rejected which will lead to undefined state, where user won't be able to operate on the lib at all.